File: core/src/main/java/org/ecos/reader/core/io/files/ZipNavigatorService.java
package org.ecos.reader.core.io.files;

import org.ecos.reader.core.io.files.exceptions.DoNoExistsException;

import reader.ecos.org.core.io.files.exceptions.UnknownError;

@SuppressWarnings("unused")
public interface ZipNavigatorService {
    FileCollection getFilesInside(ZipFile file) throws DoNoExistsException, UnknownError;
}

File: core/src/main/kotlin/org/ecos/reader/core/io/files/ZipNavigatorService.kt
package org.ecos.reader.core.io.files

import org.ecos.reader.core.io.files.exceptions.DoNoExistsException

import org.ecos.reader.core.io.files.exceptions.UnknownError

interface ZipNavigatorService {
    @Throws(DoNoExistsException::class, UnknownError::class)
    fun getFilesInside(file: ZipFile): FileCollection
}